Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.ts

```diff
@@ -0,0 +1,32 @@
+export {}
```

Comment:
Add a comment mentioning the algorithm name.

Reply:
@atarax665  Actually in typescript we just cannot take it that way some changes need to be done in html file to do that because it basically deals with the browser window so in order to take different inputs we can directly change the values.

Reply:
@shivanidalmia1203 Do this and then squash your commits